recipe/meta.yaml

```diff
@@ -57,3 +57,4 @@ extra:
     - inducer
     - jakirkham
     - yesimon
+    - isuruf
```

Comment:
This should *kind of* be a separate PR, right? :)

Reply:
Yeah. sorry about that.